client/Routes.js

```diff
@@ -40,7 +40,7 @@ class Routes extends Component {
             <Route path="/" exact component={Landing} />
             <Route path="/login" component={Login} />
             <Route path="/signup" component={Signup} />
-            <Route path="/guest" component={AllProducts} />
+            {/* <Route path="/guest" component={AllProducts} /> */}
             <Route path="/products" exact component={AllProducts} />
```

Comment:
Should this route be commented out?

Reply:
Yes. That can be removed. it is an old route that i kept just in case